e2e/components/data-table/data-table.ts

```diff
@@ -125,16 +125,13 @@ export class DataTable extends Component {
     return await this.getSortedColumnHeader().getText();
   }
 
-  async getSortingOrder() {
+  async getSortingOrder(): Promise<string> {
     const str = await this.getSortedColumnHeader().getAttribute('class');
     if (str.includes('asc')) {
       return 'asc';
     }
-    else {
-      if (str.includes('desc')) {
-        return 'desc';
-      }
-    }
+
+    return 'desc';
```

Comment:
can we keep the "else" block and return 'none' at the end ?
`if (str.includes('asc')) {
      return 'asc';
    }
    else {
      if (str.includes('desc')) {
        return 'desc';
      }
    }
    return 'none';`